### PR TITLE
Twig integration

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,3 +61,15 @@ This bundle also provides console commands to interact with rokka.io. If you onl
 install the [rokka PHP CLI tool](https://github.com/rokka-io/rokka-client-php-cli) as stand-alone phar.
 
 See the [official documentation](https://rokka.io/documentation) for further information on how to use rokka.
+
+## Twig
+
+The rokka twig extension is automatically included. For the available functions see the [README of that package](https://github.com/rokka-io/rokka-client-php-twig).
+
+## Moving from imagine
```
Comment (dbu) [inline README.md]: s/imagine/LiipImagineBundle

File: README.md [hunk 2]
```diff
@@ -61,3 +61,15 @@ This bundle also provides console commands to interact with rokka.io. If you onl
...
+
+If you use imagine and want to switch to rokka, create a rokka stack for each imagine filter you have.
+And then replace  `imagine_filter` with `rokka_stack_url` in your twig templates (maybe also state the format, if you don't want jpg).
+
+If you're images are not stored on the file system, then it needs some more adjustments. See also the twig extension for info about that.
```
Comment (dbu) [inline README.md]: s/you're/your/ 

then it needs some more adjustments. => then you need to do some further adjustments. (active voice is better)

See also the twig extension for info about that. => The twig extension readme explains how to load images from other sources.

File: composer.json [hunk 3]
```diff
@@ -31,8 +31,9 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "rokka/client": "^1.2.0",
+        "rokka/client": "^1.3.1",
         "rokka/client-cli": "^1.6.1",
+        "rokka/twig": "dev-develop as 0.0.2-dev",
```
Comment (dbu) [inline composer.json]: i got installed 0.0.3. should we require `0.0.3@dev`? or test a bit more and then release 1.0.0 of the twig thing?

**Review by dbu — COMMENTED** [review]
very nice! i only found some nitpicks for the documentation.

File: composer.json (hunk 3 above)
Comment (chregu) [reply composer.json]: i changed that to ^0.0.3 for now

File: README.md (hunk 2 above)
Comment (chregu) [reply README.md]: fixed. thanks

File: README.md (hunk 1 above)
Comment (chregu) [reply README.md]: fixed

File: composer.json (hunk 3 above)
Comment (chregu) [reply composer.json]: would be great, if someone would test it a little bit in a real world project, then I'm happy to release 1.0.0 of the twig bundle and merge this. OTOH, we can also just be adventurous and release 1.0.0 anyway ;)

**Comment by dbu** [comment]
oops, i notice we left this open since forever. should we merge it?

**Comment by chregu** [comment]
Sure. But there are merge conflicts currently

**Comment by dbu** [comment]
rebased and solved the conflict

**Comment by dbu** [comment]
is rokka/twig used anywhere? if so i suggest we tag 1.0.0 and reference that here, rather than a dev version which could break at any point.

**Comment by dbu** [comment]
ups. messed up the rebase :see_no_evil: 

adding additional config in #20

File: composer.json (hunk 3 above)
Comment (dbu) [reply composer.json]: done

**Pull request merged**